src/classes/QueryBuilder.js

```diff
@@ -1,13 +1,18 @@
 class QueryBuilder {
-  constructor (DB_CREDENTIALS, Logger) {
+  constructor (config, Logger) {
     /**
      * The knex query builder.
      * @private
      * @type    {Function}
      */
     this._knex = require('knex')({
       client: 'mysql',
-      connection: DB_CREDENTIALS,
+      connection: {
+        user: config.DB_USER,
+        database: config.DB_NAME,
+        password: config.DB_PASS,
+        host: '127.0.0.1'
+      },
```

Comment:
Can you just get these vars from `process.env` in this file? there is no need to pass config down here if the environment variables are always available.